libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1456,6 +1468,15 @@ bool AP_Mission::mission_cmd_to_mavlink_int(const AP_Mission::Mission_Command& c
         packet.param3 = cmd.content.cam_trigg_dist.trigger; // when enabled, camera triggers once immediately
         break;
 
+    case MAV_CMD_OBLIQUE_SURVEY:
+        packet.param1 = cmd.content.campos_cmd.distance;
+        packet.param3 = cmd.content.campos_cmd.interval;
+        packet.param4 = cmd.content.campos_cmd.positions;
+        packet.x = cmd.content.campos_cmd.roll;
+        packet.y = cmd.content.campos_cmd.pitch;
+        packet.frame = MAV_FRAME_MISSION;               // for compatibility with QGC
```

Comment:
Maybe this should be in an else at the end, but I didn't want to change the behavior of existing commands. For reference it would be an else to go into if the commands do not have location information:
https://github.com/ArduPilot/ardupilot/blob/88895f9b13a3b55ce3d088bfcdbd714029ff03ca/libraries/AP_Mission/AP_Mission.cpp#L1540